exporter/lambda/cross_region_import_replication.py

```diff
@@ -71,7 +71,7 @@ def _lambda_handler():
             )
         elif "No updates are to be performed." in message:
             print("No updates are to be performed.")
-        elif "UPDATE_IN_PROGRESS" in message:
+        elif message in ("UPDATE_IN_PROGRESS", "UPDATE_COMPLETE_CLEANUP_IN_PROGRESS"):
```

Comment:
Did you test this? Because I'm not sure this it going to work as `message` is a lot more stuff than just these small `StackStatus`.

Reply:
my shit memory forgot it wasn't the whole part of the message